src/parser/sparqlParser/generated/SparqlAutomatic.g4

```diff
@@ -917,20 +917,20 @@ PN_LOCAL_ESC :
 
 fragment
 PN_CHARS_BASE
-    : 'A'..'Z'
-    | 'a'..'z'
-    | '\u00C0'..'\u00D6'
-    | '\u00D8'..'\u00F6'
-    | '\u00F8'..'\u02FF'
-    | '\u0370'..'\u037D'
-    | '\u037F'..'\u1FFF'
-    | '\u200C'..'\u200D'
-    | '\u2070'..'\u218F'
-    | '\u2C00'..'\u2FEF'
-    | '\u3001'..'\uD7FF'
-    | '\uF900'..'\uFDCF'
-    | '\uFDF0'..'\uFFFD'
-    // not supported by antlr4 | '\U00010000'..'\U000EFFFF'
+    : [A-Z]
+    | [a-z]
+    | [\u00C0-\u00D6]
+    | [\u00D8-\u00F6]
+    | [\u00F8-\u02FF]
+    | [\u0370-\u037D]
+    | [\u037F-\u1FFF]
+    | [\u200C-\u200D]
+    | [\u2070-\u218F]
+    | [\u2C00-\u2FEF]
+    | [\u3001-\uD7FF]
+    | [\uF900-\uFDCF]
+    | [\uFDF0-\uFFFD]
+    | [\u{00010000}-\u{000EFFFF}]
```

Comment:
@joka921 This is the core "fix", the older ANTLR 3 syntax with `'\uABCD'..'\uBCDF'` didn't support the higher unicode ranges, but the v4 syntax does (at least according to [this comment on stackoverflow](https://stackoverflow.com/a/56323017))

Reply:
I also decided to adjust all the others in this fragment too for consistency, but now all this is inconsistent with regards to all other fragments

Reply:
Nice, thanks for finding that out,